app/data/newsroom.json

```diff
@@ -224,6 +224,20 @@
       "img": "/static/thumbnail.png",
       "filename": "lead-maintainer-report-march-2020"
     },
+    {
+      "title": "Bitcoin Cash Node 2020: planos para atualização de maio e adiante.",
+      "date": "04 March 2020",
+      "author": "phabulu",
+      "img": "/static/thumbnail.png",
+      "filename": "Bitcoin-Cash-Node-2020-planos-para-atualização-de-maio-e-adiante"
```

Comment:
This article is already merged & present on the newsroom
(I think you can remove this Portuguese section from this PR)